pysteps/blending/linear_blending.py

```diff
@@ -208,3 +258,70 @@ def forecast(
         R_blended = precip_nowcast
 
     return R_blended
+
+
+def get_ranked_salience(precip_nowcast, precip_nwp):
+    """
+    Calculate ranked salience
+
+    Args:
+    precip_nowcast: array_like
+      Array of shape (m,n) containing the extrapolated precipitation field at a specified timestep
+    precip_nwp: array_like
+      Array of shape (m,n) containing the NWP fields at a specified timestep
+
+    Returns:
+      Array of shape (m,n) containing ranked salience
+    """
+
+    # calcutate normalized intensity
```

Comment:
here we assume that the input fields are bounded at zero, correct (no negative values)?